packages/suite/src/components/suite/modals/ReduxModal/CardanoWithdrawModal.tsx

```diff
@@ -0,0 +1,156 @@
+import { FormattedMessage } from 'react-intl';
+
+import styled from 'styled-components';
+
+import { Button, H2, Icon, Link, NewModal } from '@trezor/components';
+import { borders, typography } from '@trezor/theme';
+
+import { useSelector } from 'src/hooks/suite/useSelector';
+import { useCardanoStaking } from 'src/hooks/wallet/useCardanoStaking';
+
+import { HiddenPlaceholder } from '../../HiddenPlaceholder';
+
+// eslint-disable-next-line local-rules/no-override-ds-component
+export const StyledH2 = styled(H2)`
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    margin-bottom: 5px;
+`;
```

Comment:
overrides like this are not allowed anymore right? cc @jvaclavik 
edit: haha, now I noticed the eslint-disable-next-line comment 👿